src/components/NewLocationPage/ChartFooter/MetricChartFooter.tsx

```diff
@@ -28,56 +24,106 @@ export interface ShareButtonProps {
   showEmbedButton?: boolean;
 }
 
+interface DialogProps {
+  open: boolean;
+  closeDialog: () => void;
+  openDialog: () => void;
+  modalContent: MetricModalContent;
+  modalHeader: string;
+}
+
+const ShareButtonBlock: React.FC<ShareButtonProps> = ({
+  ...shareButtonProps
+}) => {
+  return (
+    <ButtonContainer>
+      <ShareButtons {...shareButtonProps} />
+    </ButtonContainer>
+  );
+};
+
+const MetricModal: React.FC<DialogProps> = ({
+  open,
+  closeDialog,
+  openDialog,
+  modalContent,
+  modalHeader,
+}) => {
+  const dialogProps = {
+    open,
+    closeDialog,
+    header: modalHeader,
+    links: [
+      {
+        cta: 'Learn more',
+        url: modalContent.learnLink,
+      },
```

Comment:
We should add a good `aria-label` for this link, although given that the modal covers a single metric, a screen reader user will probably know that the link is to Lean more about that metric.